ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -27,6 +25,9 @@ def enable_appsec_rc():
     from ddtrace import tracer
 
     if _appsec_rc_features_is_enabled():
+        from ddtrace.internal.remoteconfig import RemoteConfig
```

Comment:
I'm not sure to understand why it wouldn't work

Reply:
Since `_remoteconfiguration.py` is an internal module, I don't think users should be impacted by this change.

@gnufede do we expect users to import RemoteConfig from this module? 
ex: `from ddtrace.appsec._remoteconfiguration import RemoteConfig`

Reply:
Sorry it took me this long, I just double checked and it works fine, I stand corrected